functests/utils/consts.go

```diff
@@ -24,7 +24,15 @@ var NodesSelector string
 // ProfileNotFound is true when discovery mode is enabled and no valid profile was found
 var ProfileNotFound bool
 
+// PerformanceOperatorNamespace contains the name of the performance operator namespace
+var PerformanceOperatorNamespace string
+
 func init() {
+	PerformanceOperatorNamespace = os.Getenv("PERFORMANCE_OPERATOR_NAMESPACE")
```

Comment:
I'd use `LookupEnv`

Reply:
no strong opinions here, hence I will do that. But what's the benefit in this specific case? Could you please elaborate for my understanding?